test/src/measure.py

```diff
@@ -132,45 +137,52 @@ def estimated_boottime(num_vms: int) -> float:
 
     @contextmanager
     def virtual_machine(self, interface: Interface) -> Iterator[Guest]:
+        """
+        Creates a guest virtual machine
+        """
+
         # host: inital cleanup
 
         debug('Initial cleanup')
         try:
             self.host.kill_guest()
         except Exception:
             pass
-        self.host.cleanup_network()
 
-        # host: set up interfaces and networking
+        try:
```

Comment:
why is this a try block? If any of this fails, the measurement should fail as well. If you want to catch failure for debugging purposes, you can run with `python3 -m pdb` to handle or ignore exceptions manually.